sunnypilot/selfdrive/controls/lib/lane_turn_desire.py

```diff
@@ -4,25 +4,36 @@
 This file is part of sunnypilot and is licensed under the MIT License.
 See the LICENSE.md file in the root directory for more details.
 """
-from cereal import custom
+from cereal import custom, log
 
 from openpilot.common.constants import CV
 from openpilot.common.params import Params
 
 LANE_CHANGE_SPEED_MIN = 20 * CV.MPH_TO_MS
 
+TURN_STATE = {
+  'OFF': 0,
+  'NUDGE': 1,
+  'NUDGELESS': 2
+}
+
+TURN_DESIRES = {
+  custom.TurnDirection.none: log.Desire.none,
+  custom.TurnDirection.turnLeft: log.Desire.turnLeft,
+  custom.TurnDirection.turnRight: log.Desire.turnRight,
+}
+
 
 class LaneTurnController:
-  def __init__(self, desire_helper):
-    self.DH = desire_helper
+  def __init__(self):
     self.turn_direction = custom.TurnDirection.none
     self.params = Params()
     self.lane_turn_value = float(self.params.get("LaneTurnValue", return_default=True)) * CV.MPH_TO_MS
     self.param_read_counter = 0
-    self.enabled = self.params.get_bool("LaneTurnDesire")
+    self.lane_turn_type = int(self.params.get("LaneTurnDesire", return_default=True))
```

Comment:
If the param type is `INT`, you don't need to convert it to `int` again. If lint is failing, try setting a type hint for the attributes.